docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -147,7 +149,7 @@ You might not want to store all 20 fields but pick and choose what suits your re
 
 Then, we can write a function that gets the tweet data for each tweet returned in `search_results` generator object.
 
-```python
+```bash linenums="1" hl_lines="5-12" title="python"
```

Comment:
the language is python and remove highlights?

Reply:
done